tutorials/hist/ContourList.C

```diff
@@ -115,23 +116,23 @@ TCanvas *ContourList(){
    2, -2, 2, 2, 0, 6.5);
 
    hr->Draw();
-   Double_t x0, y0, z0;
+   Double_t xval0, yval0, zval0;
```

Comment:
Was this renaming needed? This should not have caused a compilation error. If it did I'd like to know; this would mean we have a different issue that we're not aware of yet...